WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.java

```diff
@@ -0,0 +1,178 @@
+package org.wordpress.android.ui.mysite;
+
+import android.os.Bundle;
+import android.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import org.wordpress.android.R;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.models.Blog;
+import org.wordpress.android.ui.ActivityLauncher;
+import org.wordpress.android.util.GravatarUtils;
+import org.wordpress.android.util.StringUtils;
+import org.wordpress.android.widgets.WPNetworkImageView;
+import org.wordpress.android.widgets.WPTextView;
+
+public class MySiteFragment extends Fragment {
+
+    private WPNetworkImageView mBlavatarImageView;
+    private WPTextView mBlogTitleTextView;
+    private WPTextView mBlogSubtitleTextView;
+    private int mBlavatarSz;
+
+    private Blog mBlog;
+
+    public static MySiteFragment newInstance() {
+        MySiteFragment fragment = new MySiteFragment();
+        Bundle args = new Bundle();
```

Comment:
Since there aren't any arguments, this can be simplified to just `return new MySiteFragment();`